pkg/kubectl/describe.go

```diff
@@ -1447,7 +1447,7 @@ type HorizontalPodAutoscalerDescriber struct {
 }
 
 func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (string, error) {
-	hpa, err := d.client.Extensions().HorizontalPodAutoscalers(namespace).Get(name)
```

Comment:
It's interesting question. @bgrant0607 do we have any policy here?
It's not negligible that API server should support previous versions of api for some time to allow users migration. The question is whether API server should support further versions of api which is of course impossible but this is actually the case.

If apiserver 1.1 should work with kubectl 1.2 should also apiserver 1.1 work with kubelet 1.2?